ui/about_window.h

```diff
@@ -34,13 +34,30 @@ class AboutDialog : public QDialog
     AboutDialog(QWidget *parent = 0);
 
 private:
-    QIcon          *m_icon;                  // The Dive icon
-    QLabel         *m_icon_label;            // A label widget to hold the icon.
-    QLabel         *m_build_information;     // The build info and version string.
-    QLabel         *m_third_party_licenses;  // The NOTICE file text.
-    QPlainTextEdit *m_license_notice;        // An edit box to hold the NOTICE text.
-    QPushButton    *m_close_button;          // A button to close the dialog.
-    QHBoxLayout *m_version_layout;  // The version layout is Icon + Build Info arranged horizontally
-    QHBoxLayout *m_button_layout;   // The button layout is stretch + Close Button
-    QVBoxLayout *m_main_layout;     // The main layout Version + Text + Licenses arranged vertically
+    void CreateHeaderLayout();
```

Comment:
Make those function return a layout, and you don't need most of the private class members.